content/_partials/video.html.haml

```diff
@@ -0,0 +1,19 @@
+- video_id = rand(1000)
+- allow_attribs = "accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+
+:css
+  .video_#{video_id} {
+    position: relative;
+    padding-bottom: 56.25%; /* 16:9 */
+    height: 0;
+  }
+  .video_#{video_id} iframe {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+  }
+
+.video#ProjectVideo{:class => "video_#{video_id}"}
```

Comment:
```suggestion
.video{:class => "video_#{video_id}"}
```

At one point non unique IDs would cause weird stuff in browser rendering. I suspect they fixed it, but still IDs should be unique. Removing the ID since your not using it, would allow a video twice on the same page.